src/packagedcode/rpm.py

```diff
@@ -38,27 +38,33 @@
 #  http://www.faqs.org/docs/artu/ch05s02.html#id2906931%29.)
 #  http://code.activestate.com/recipes/436229-record-jar-parser/
 
-RPM_TAGS = ('name',
-           'version',
-           'release',
-           'summary',
-           # the full description is often a long text
-           'description',
-           'distribution',
-           'vendor',
-           'license',
-           'packager',
-           'group',
-           'patch',
-           'url',
-           'os',
-           'arch',
-           'source_rpm',
-           'source_package',
-           'dist_url',
-           'bin_or_src',
-           )
-
+RPM_TAGS = (
+    'name',
+    'epoch',
+    'version',
+    'release',
+    'arch',
+    'os',
+    'summary',
+    # the full description is often a long text
+    'description',
+    'distribution',
+    'vendor',
+    'license',
+    'packager',
+    'group',
+    'patch',
+    'url',
+    'source_rpm',
+    'source_package',
+    'dist_url',
+    'bin_or_src',
+)
+
+
+class EnhancedRpmPackage(RpmPackage):
```

Comment:
Why this subclass?
You want rather to update the RpmPackage class, eventually moving it there but not create an EnhancedRpmPackage subclass